smsgateway/utils.py

```diff
@@ -20,7 +22,18 @@ def check_cell_phone_number(number):
     return format_number(parsed_number, PhoneNumberFormat.E164)
 
 
-def truncate_sms(text, max_length=160):
+def get_max_msg_length():
+    account = get_account()
+    try:
+        return account['max_msg_length']
+    except KeyError:
+        return
+
+
+def truncate_sms(text, max_length=get_max_msg_length() or None):
```

Comment:
is there a reason to do the `or None` just to fallback to 160 two lines lower? just wondering...